<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Mdt::SerialPort::SettingsDialog</class>
 <widget class="QDialog" name="Mdt::SerialPort::SettingsDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>429</width>
    <height>329</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Serial port settings</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QGroupBox" name="selectPortBox">
       <property name="title">
        <string>Select serial port</string>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_2">
          <item>
           <widget class="QComboBox" name="serialPortListBox"/>
          </item>
          <item>
           <widget class="QToolButton" name="refreshSerialPortListButton">
            <property name="toolTip">
             <string>Refresh serial port list</string>
            </property>
            <property name="text">
             <string>Refresh</string>
            </property>
            <property name="icon">
             <iconset theme="view-refresh"/>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QFormLayout" name="formLayout">
          <item row="0" column="0">
           <widget class="QLabel" name="descriptionLabel">
            <property name="text">
             <string>Description:</string>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QLabel" name="description">
            <property name="text">
             <string/>
            </property>
           </widget>
          </item>
          <item row="1" column="0">
           <widget class="QLabel" name="manufacturerLabel">
            <property name="text">
             <string>Manufacturer:</string>
            </property>
           </widget>
          </item>
          <item row="1" column="1">
           <widget class="QLabel" name="manufacturer">
            <property name="text">
             <string/>
            </property>
           </widget>
          </item>
          <item row="2" column="0">
           <widget class="QLabel" name="serialNumberLabel">
            <property name="text">
             <string>Serial number:</string>
            </property>
           </widget>
          </item>
          <item row="2" column="1">
           <widget class="QLabel" name="serialNumber">
            <property name="text">
             <string/>
            </property>
           </widget>
          </item>
          <item row="3" column="0">
           <widget class="QLabel" name="locationLabel">
            <property name="text">
             <string>Location:</string>
            </property>
           </widget>
          </item>
          <item row="3" column="1">
           <widget class="QLabel" name="location">
            <property name="text">
             <string/>
            </property>
           </widget>
          </item>
          <item row="4" column="0">
           <widget class="QLabel" name="vidLabel">
            <property name="text">
             <string>Vendor ID:</string>
            </property>
           </widget>
          </item>
          <item row="4" column="1">
           <widget class="QLabel" name="vid">
            <property name="text">
             <string/>
            </property>
           </widget>
          </item>
          <item row="5" column="0">
           <widget class="QLabel" name="pidLabel">
            <property name="text">
             <string>Product ID:</string>
            </property>
           </widget>
          </item>
          <item row="5" column="1">
           <widget class="QLabel" name="pid">
            <property name="text">
             <string/>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <widget class="QGroupBox" name="portSettingsBox">
       <property name="title">
        <string>Port settings</string>
       </property>
       <layout class="QFormLayout" name="formLayout_2">
        <item row="0" column="0">
         <widget class="QLabel" name="baudRateLabel">
          <property name="text">
           <string>Baud rate:</string>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QComboBox" name="baudRateBox"/>
        </item>
        <item row="1" column="0">
         <widget class="QLabel" name="dataBits">
          <property name="text">
           <string>Data bits:</string>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="QComboBox" name="dataBitsBox"/>
        </item>
        <item row="2" column="0">
         <widget class="QLabel" name="parityLabel">
          <property name="text">
           <string>Parity:</string>
          </property>
         </widget>
        </item>
        <item row="2" column="1">
         <widget class="QComboBox" name="parityBox"/>
        </item>
        <item row="3" column="0">
         <widget class="QLabel" name="stopBitsLabel">
          <property name="text">
           <string>Stop bits:</string>
          </property>
         </widget>
        </item>
        <item row="3" column="1">
         <widget class="QComboBox" name="stopBitsBox"/>
        </item>
        <item row="4" column="0">
         <widget class="QLabel" name="flowControlLabel">
          <property name="text">
           <string>Flow control:</string>
          </property>
         </widget>
        </item>
        <item row="4" column="1">
         <widget class="QComboBox" name="flowControlBox"/>
        </item>
       </layout>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>Mdt::SerialPort::SettingsDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>Mdt::SerialPort::SettingsDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
